pcdsdevices/mirror.py

```diff
@@ -187,6 +187,8 @@ class OffsetMirror(Device):
                    kind='config')
     # Transmission for Lightpath Interface
     transmission = 1.0
+    # Subscription types
+    SUB_STATE = 'state'
```

Comment:
Was this really all that had to be added here? I'm surprised.

Reply:
😄

Reply:
A subscription that is never run can still be subscribed to nonetheless.